pxd_fastpath.c

```diff
@@ -455,6 +410,47 @@ static void __pxd_cleanup_block_io(struct pxd_io_tracker *head)
 	bio_put(&head->clone);
 }
 
+static void pxd_io_failover(struct work_struct *ws)
```

Comment:
Is it be possible to freeze_bdev and then trigger a switch fast/slow. That would reduce the amount of work here rt?

Reply:
Thats what pxd_suspend_io and pxd_resume_io api calls do.